java-api/src/main/java/com/sap/cloud/security/x509/X509Constants.java

```diff
@@ -0,0 +1,16 @@
+/**
+ * SPDX-FileCopyrightText: 2018-2021 SAP SE or an SAP affiliate company and Cloud Security Client Java contributors
+ * <p>
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.sap.cloud.security.x509;
+
+public final class X509Constants {
+
+	private X509Constants() {
+	}
+
+	public static final String X5T_VALIDATOR_ENABLED = "X5T_VALIDATOR_ENABLED";
```

Comment:
should not be required in that PR, same is true for ACCEPT_CALLERS_BOUND_TO_SAME_ID_SERVICE